### PR TITLE
Add ProxyJump completion

### PR DESCRIPTION
**Pull request by jmwallach**
ProxyJump has been in place since 7.3

### CODE REVIEW

File: completions/ssh [hunk 1]
```diff
@@ -44,8 +44,8 @@ _ssh_options()
         IdentityFile IdentitiesOnly IPQoS KbdInteractiveDevices KexAlgorithms
         LocalCommand LocalForward LogLevel MACs
         NoHostAuthenticationForLocalhost NumberOfPasswordPrompts
-        PasswordAuthentication PermitLocalCommand PKCS11Provider Port
-        PreferredAuthentications Protocol ProxyCommand PubkeyAuthentication
+        PasswordAuthentication PermitLocalCommand PKCS11Provider ProxyJump Port
```
Comment (scop) [inline completions/ssh]: Remove this, one addition is enough :)

File: completions/ssh [hunk 2]
```diff
@@ -215,7 +219,6 @@ _ssh()
         local configfile
         _ssh_configfile
         _known_hosts_real -a -F "$configfile" -- "$cur"
-
```
Comment (scop) [inline completions/ssh]: This whitespace change seems unrelated, please remove it.

File: completions/ssh [hunk 3]
```diff
@@ -116,6 +116,10 @@ _ssh_suboption()
         Ciphers)
             _ssh_ciphers "$2"
             ;;
+        ProxyJump)
+            _known_hosts_real -- "$cur"
```
Comment (scop) [inline completions/ssh]: Shouldn't `-a` be used for `_known_hosts_real` here?

File: completions/ssh (hunk 1 above)
Comment (jmwallach) [reply completions/ssh]: I fixed the issues as mentioned inline and was preparing the -J option when I noticed that ProxyJump accepts comma-separated lists. Since _ssh_suboptions is not supposed to accept these arguments I am going to move the addition to my local config and not push the update.

**Comment by scop** [comment]
Not supporting comma separated values is better than nothing, so I've pushed a change finishing support for that.

**Comment by jmwallach** [comment]
Glad to have inspired someone to do the work :)
